qiskit/transpiler/synthesis/solovay_kitaev/solovay_kitaev_plugin.py

```diff
@@ -0,0 +1,135 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+A Solovay-Kitaev synthesis plugin to Qiskit's transpiler.
+"""
+
+from qiskit.converters import circuit_to_dag
+from qiskit.transpiler.passes.synthesis.plugin import UnitarySynthesisPlugin
+
+from .solovay_kitaev import SolovayKitaev
+from .generate_basis_approximations import generate_basic_approximations
+
+
+class SolovayKitaevSynthesisPlugin(UnitarySynthesisPlugin):
```

Comment:
Similar comment to the transformation pass I think we should put this with the other unitary synthesis plugins in `qiskit/transpiler/passes/synthesis`.

Reply:
done in [961babc](https://github.com/Qiskit/qiskit-terra/pull/5657/commits/961babcab801cf6f1c006ccc6b3363ea7d146832)